index.js

```diff
@@ -0,0 +1,2 @@
+require('./format-as-currency');
```

Comment:
merged! upgrade to v3 and everything should work.

if you want to get rid of that warning, `require('format-as-currency/src/format-as-currency')` should do the trick. also update the docs to reflect this.